armi/cli/clone.py

```diff
@@ -29,10 +29,8 @@ class CloneArmiRunCommandBatch(EntryPoint):
 
     def addOptions(self):
         for settingName in self.cs.keys():
-            # verbosity and branchVerbosity already have command line options in the default parser
-            # adding them again would result in an error from argparse.
-            if settingName not in ["verbosity", "branchVerbosity"]:
```

Comment:
I love that we're ditching this "special case".   

I expected to see this moved down into the `createOptionFromSetting()` method, but it's not there.  Where did that go?

Don't get me wrong, I think the code looks much cleaner this way.

Reply:
This is captured by the try/except in that method. Basically the reason we skipped this is because it was "already added" but catching the ArgumentError when trying to add_argument will just skip an optional argument (or a setting in this case) that already exists. The parser doesn't have any public attributes to see what is already an argument without iterating through it's ._actions attribute so I thought it was cleaner to just catch an exception versus trying to write some tooling to check if the action was there already. I am concerned that if they rename that attribute in the future then it could break our stuff. Let me know if want me to explicitly search down that actions list though.